documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -9,57 +13,92 @@
-serialize each message key and value into a binary form using the configured https://kafka.apache.org/documentation/#connect_running[_converters_],
-and finally write each messages into the correct Kafka topic.
+{prodname} connectors work with the Kafka Connect framework to capture changes in databases and generate change event records. The Kafka Connect workers then apply any configured transformations to each of the messages generated by the connector, serialize each message key and value into a binary form by using the configured link:https://kafka.apache.org/documentation/#connect_running[_converters_], and write each message into the correct Kafka topic.
 
-The converters can either be specified in the Kafka Connect worker configuration,
-in which case the same converters are used for all connectors deployed to that worker's cluster.
-Alternatively, they can be specified for an individual connector.
-Kafka Connect comes with a _JSON converter_ that serializes the message keys and values into JSON documents.
-The JSON converter can be configured to include or exclude the message schema using the (`key.converter.schemas.enable` and `value.converter.schemas.enable`) properties.
-Our {link-prefix}:{link-tutorial}[tutorial] shows what the messages look like when both payload and schemas are included, but the schemas make the messages very verbose.
-If you want your messages serialized with JSON, consider setting these properties to `false` to exclude the verbose schema information.
+You can specify converters in several ways: 
 
-Alternatively, you can serialize the message keys and values using https://avro.apache.org/[Apache Avro].
-The Avro binary format is extremely compact and efficient, and Avro schemas make it possible to ensure that the messages have the correct structure.
-Avro's schema evolution mechanism makes it possible to evolve the schemas over time,
-which is essential for {prodname} connectors that dynamically generate the message schemas to match the structure of the database tables.
-Over time, the change events captured by {prodname} connectors and written by Kafka Connect into a topic may have different versions of the same schema,
-and Avro serialization makes it far easier for consumers to adapt to the changing schema.
+* In the Kafka Connect worker configuration. 
++
+In this case, the same converters are used for all connectors that are deployed to that worker's cluster.
 
-== The Apicurio API and Schema Registry
+* For an individual connector. 
++
+Kafka Connect comes with a _JSON converter_ that serializes message keys and values into JSON documents. You can configure the JSON converter to include or exclude the message schema by specifying the `key.converter.schemas.enable` and `value.converter.schemas.enable` properties. The {prodname} {link-prefix}:{link-tutorial}[tutorial] shows what the messages look like when both payload and schemas are included. 
```

Comment:
I feel like this and the following two parameters apply to both Kafka worker or individual connector setups and probably shouldn't be indented under the second option.

Reply:
Sure. I removed the plus signs that indent those paragraphs. Fixed in the next commit.